src/condition.cpp

```diff
@@ -1794,9 +1794,14 @@ bool ConditionLight::setParam(ConditionParam_t param, int32_t value)
 	}
 
 	switch (param) {
-		case CONDITION_PARAM_LIGHT_LEVEL:
-			lightInfo.level = value;
+		case CONDITION_PARAM_LIGHT_LEVEL: {
+			if (value < 1) {
+				std::cout << "[ConditionLight::setParam] trying to set invalid light value: " << value
+				          << " defaulting to 1" << std::endl;
+			}
+			lightInfo.level = std::max(1, value);
```

Comment:
since lightinfo.level is a uint8_t and value is an int32_t, maybe we should go ahead and add some safety here and use clamp instead?